country-info.csv

```diff
@@ -150,3 +150,4 @@ York University,canada
 Zhejiang University,asia
 Bielefeld University,europe
 KTH Royal Institute of Technology,europe
+Universidade Federal de Pernambuco,southamerica
```

Comment:
For symmetry with other Brazilian institutions, please abbreviate (e.g., UFP). Thanks.

Reply:
@damorim The institutional name should also be abbreviated in csrankings.csv.